### PR TITLE
Add coverage report

### PR DESCRIPTION
**Pull request by gandarez**
This PR replaces the inactive and outdated https://github.com/wakatime/wakatime-cli/pull/294.

### CODE REVIEW

File: .github/workflows/on_push_protected_branches.yml [hunk 1]
```diff
@@ -30,6 +30,11 @@ jobs:
       - 
         name: "Linter"
         run: make lint
+      -
+        name: "Send coverage"
+        uses: shogo82148/actions-goveralls@v1
+        with:
+          path-to-profile: coverage.out
```
Comment (dron22) [inline .github/workflows/on_push_protected_branches.yml:37]: I guess we need to generate the coverage profile, when running tests, right? Compare: https://github.com/mattn/goveralls/blame/master/README.md#L65

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # WakaTime CLI
 
+| branch  | build | coverage |
```
Comment (dron22) [inline README.md:3]: The coverage of `develop` branch is mainly for internal purposes and I guess we don't need to have it on top of the README. Let's keep it clean. Badge for master would be enough here I guess.

File: .github/workflows/on_push_protected_branches.yml (hunk 1 above)
Comment (gandarez) [reply .github/workflows/on_push_protected_branches.yml:37]: exactly and the coverage profile is being generated by make lint, see the changes on MAKEFILE.

File: README.md (hunk 2 above)
Comment (gandarez) [reply README.md:3]: As we want to `develop` branch for testing and alpha releases I believe having it at the top would be useful for people who want to make sure build is passing. If we add it why not to also have its coverage reported and exposed?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+| ---     | ---   | ---      |
+| develop | ![Build develop](https://img.shields.io/github/workflow/status/wakatime/wakatime-cli/Create%20Release/develop) | [![Coverage Status](https://coveralls.io/repos/github/wakatime/wakatime-cli/badge.svg?branch=develop)](https://coveralls.io/github/wakatime/wakatime-cli?branch=develop)
```
Comment (dron22) [inline README.md:5]: `Create Release` workflow actually does not represent the build process.

Comment (gandarez) [reply README.md:5]: What about having `Create Release` as **Unit Tests Status** and `Build and upload release assets` as **Build Status**?

Comment (dron22) [reply README.md:5]: Yeah, this sounds like the right way :+1:

**Review by dron22 — APPROVED** [review]
As you removed the `develop` branch stats, we don't really need the markdown table anymore, right? We could have the badges without the table on the top, like most repos do it.

Not explicitly mentioning master branch would be alright I guess, as it is safe to assume that the stats represent the latest official release.

Approving already..

**Pull request merged**